src/foam/blob/BlobJava.js

```diff
@@ -100,4 +100,15 @@ foam.INTERFACE({
       ]
     }
   ]
+});
+
+foam.CLASS({
+  refines: 'foam.blob.IdentifiedBlob',
+
+  properties: [
+    {
+      name: 'delegate',
+      javaFactory: 'return ((BlobService) getBlobService()).find(getId());'
```

Comment:
We may have to inline this right into the main class.  I'm not sure refining javaFactory works properly.  It may clobber the regular JS factory.

Reply:
Okay. Moved